docs/imagecustomizer/api/configuration/input.md

```diff
@@ -4,18 +4,36 @@ parent: Configuration
 
 # input type
 
-Specifies the configuration for the input image.
+Specifies the configuration for the input image and associated files required for customizing.
 
 Example:
 
 ```yaml
 input:
   image:
     path: ./base.vhdx
+  verityHashes:
+    path: /input/verityhashes
+  ukis:
+    path: /input/ukis
+  bootloader:
```

Comment:
is this the bootloader itself or the bootloader configuration? I'd expect it's the configuration, correct?